### PR TITLE
doc: Add known issue for v2.5.2

### PR DESCRIPTION
**Pull request by jori-nordic**
See the issue text.

### CODE REVIEW

**Comment by umapraseeda** [comment]
FYI, @peknis

**Comment by peknis** [comment]
I will add this issue in my https://github.com/nrfconnect/sdk-nrf/pull/14185. Please close this one.

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 1]
```diff
@@ -216,6 +216,13 @@ Only secure applications can use Bluetooth LE
   When running the :ref:`bluetooth_central_dfu_smp` sample, the :kconfig:option:`CONFIG_BT_SMP` configuration must be aligned between this sample and the Zephyr counterpart (:zephyr:code-sample:`smp-svr`).
   However, security is not enabled by default in the Zephyr sample.
 
+.. rst-class:: v2-5-2
+
+DRGN-21390: The `disconnected` callback might not get called on nRF5340.
+  The Bluetooth host running on the nRF5340 application core might deadlock on disconnection.
+  This is due to a recent bugfix in the Softdevice Controller triggering a bug in the ATT queuing layer.
+
+  **Workaround:** Either disable host flow control (:kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL`) or cherry-pick commits from the upstream: `Zephyr PR #65272 <https://github.com/zephyrproject-rtos/zephyr/pull/65272>`_.
```
Comment (LuDuda) [inline doc/nrf/releases_and_maturity/known_issues.rst:225]: Can we clarify if it's enough to disable it on App or Net core, or customer needs to do it on both App and Net core.

**Review by LuDuda — COMMENTED** [review]
LGTM

File: doc/nrf/releases_and_maturity/known_issues.rst (hunk 1 above)
Comment (jori-nordic) [reply doc/nrf/releases_and_maturity/known_issues.rst:225]: It's only on the app core. This kconfig is only valid when there's a host being compiled

Comment (jori-nordic) [reply doc/nrf/releases_and_maturity/known_issues.rst:225]: The text moved to this PR: https://github.com/nrfconnect/sdk-nrf/pull/14185
Could you approve it there instead?

**Pull request closed without merging**